### ISSUE TITLE
Initial configuration for Vertical direction is not working 

### ISSUE THREAD

**Issue by NHorde** [lastboy/funnel-graph-js#31]
I bumped into your library and noticed that the vertical direction is not working (https://codepen.io/arik-test/pen/OPLMOzK) - simply set `direction: "vertical"`

Is it something that you have flagged?

**Comment by lastboy** [comment]
You are right, I checked the code, by mistake I set the direction with the wrong api on initial.
I will fix,

Thank you,
@NHorde

**Comment by lastboy** [comment]
need to add change log and update the npm version
